bundles/WebToPrintBundle/.github/ci/files/bin/console

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+ob_get_clean();
+
+if (file_exists($a = getcwd() . '/vendor/autoload.php')) {
```

Comment:
This doesn't seem to be up-to date, when I look at: https://github.com/pimcore/skeleton/blob/11.x/bin/console

Reply:
Yep, let's sync the stuff after extraction.